CHANGELOG.next.asciidoc

```diff
@@ -26,6 +26,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Modify apache/error dataset to follow ECS. {pull}8963[8963]
 - Rename many `traefik.access.*` fields to map to ECS. {pull}9005[9005]
 - Fix parsing of GC entries in elasticsearch server log. {issue}9513[9513] {pull}9810[9810]
+- Rename a few `nginx.error.*` fields to map to ECS. {pull}10007[10007]
```

Comment:
Traefik PR above was opened 2 months ago. 1000+ issues and PRs since then. :mind-blown: